examples/js/loaders/ColladaLoader.js

```diff
@@ -18,17 +18,24 @@ THREE.ColladaLoader.prototype = {
 	load: function ( url, onLoad, onProgress, onError ) {
 
 		var scope = this;
-
-		var path = THREE.Loader.prototype.extractUrlBase( url );
-
+		
+                scope.path = scope.path === undefined ? THREE.Loader.prototype.extractUrlBase( url ) : scope.path; 
```

Comment:
I would make this:

```js
var path = scope.path === undefined ? THREE.Loader.prototype.extractUrlBase( url ) : scope.path;
```

Otherwise consider this case:

```js
var loader = new THREE.ColladaLoader();
loader.load('path/to/first/model.dae', () => ...);
loader.load('other/path/to/second/model.dae', () => ...);
```

The first model will modify `scope.path` and break the second model.

Reply:
If we do it like this, we also handle the path for `textureLoader` correctly, see

https://github.com/cnspaha/three.js/blob/18b700af3c3e50f82a8112f54061af212db8011e/examples/js/loaders/ColladaLoader.js#L3367

@cnspaha The `path` value at this place is passed as a parameter of `parse()`. With your current implementation, this value isn't set correctly.

Reply:
I think it would be the best to handle this in the same way like `GLTFLoader`.

Reply:
@donmccurdy 
I get your point. 
Just saw, that I pretty messed up this one...
I like your idea, thanks again for your input ;)

@Mugen87 
The suggestion of @donmccurdy  is exactly what the GLTFLoader does, isn't it?